sw/device/tests/pmp_sanitytest.c

```diff
@@ -0,0 +1,85 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/runtime/pmp.h"
+
+#include "sw/device/lib/handler.h"
+#include "sw/device/lib/irq.h"
+#include "sw/device/lib/runtime/check.h"
+#include "sw/device/lib/runtime/hart.h"
+#include "sw/device/lib/testing/test_main.h"
+#include "sw/device/lib/testing/test_status.h"
+
+/**
+ * PMP regions that are used for load/store and execution permission violation
+ * tests.
+ *
+ * "Volume II: RISC-V Privileged Architectures V20190608-Priv-MSU-Ratified",
+ * "3.6 Physical Memory Protection", "Address Matching":
+ *
+ * "If PMP entry i’s A field is set to TOR, the entry matches any address y
+ * such that pmpaddri−1 ≤ y < pmpaddri. If PMP entry 0’s A field is set to TOR,
+ * zero is used for the lower bound, and so it matches any address
+ * y < pmpaddr0."
+ *
+ * To prtotect an address range that starts above 0 address, the first region
+ * we can use is 1.
+ */
+#define PMP_LOAD_STORE_TOR_REGION_ID 1
+
+/**
+ * The buffer that is used for load/store access violation test.
+ */
+__attribute__((aligned(4))) static volatile char pmp_load_store_test_data[2048];
```

Comment:
Can you stick a `//` after the `__attribute__` to force clang-format to format this somewhat more sanely? (I think this should cause clang-format to wrap things a bit more nicely...)